lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -500,6 +525,38 @@ def parse_severity(severity_str)
       'DEFAULT'
     end
 
+    def handle_container_metadata(record, entry)
+      labels = {}
+      fields = %w(namespace_id namespace_name pod_id pod_name container_name)
+      fields.each do |field|
+        field_to_label(record['kubernetes'], field, labels,
+                       "#{CONTAINER_SERVICE}/#{field}")
+        field_to_label(record['kubernetes'], field, labels,
+                       "#{CONTAINER_SERVICE}/#{field}")
+        field_to_label(record['kubernetes'], field, labels,
+                       "#{CONTAINER_SERVICE}/#{field}")
+        field_to_label(record['kubernetes'], field, labels,
+                       "#{CONTAINER_SERVICE}/#{field}")
+        field_to_label(record['kubernetes'], field, labels,
+                       "#{CONTAINER_SERVICE}/#{field}")
+      end
+      # Append label/ to all user-defined labels' keys.
+      if record.key?('labels')
+        record['kubernetes']['labels'].each do |key, value|
+          labels["label/#{key}"] = value
+        end
+      end
+      record.delete('kubernetes')
+      record.delete('docker')
```

Comment:
Add a comment explaining why we're deleting this without consuming its contents.